apps/arena/lib/arena/game_updater.ex

```diff
@@ -824,12 +824,22 @@ defmodule Arena.GameUpdater do
     {bushes, last_id} = initialize_bushes(config.map.bushes, last_id)
     {pools, last_id} = initialize_pools(config.map.pools, last_id)
 
+    zone_shrinking_center_position =
+      random_position_in_map(
+        0.0,
```

Comment:
What's this zero hardcoded value?

Reply:
That function has weird parameters but that `0.0` represents the center of the circle that's going to be created to select a random position within that circle.